lib/cc/engine/analyzers/engine_config.rb

```diff
@@ -20,13 +23,29 @@ def include_paths
         end
 
         def languages
-          config.fetch("languages", {})
+          config.fetch("languages", ["ruby"])
         end
 
         def concurrency
           config.fetch("config", {}).fetch("concurrency", 2).to_i
         end
 
+        def checks
+          config.fetch("config", {}).fetch("checks", {})
+        end
+
+        def enabled?(check)
+          checks.fetch(check, {}).fetch("enabled", "false").to_s.casecmp("true").zero?
```

Comment:
1. The values in the config will be real booleans, not strings. (Bools in the YAML `config` subkey are strings today because of weirdness in `cc-yaml`, but that has no bearing on the new stuff.)

2. Shouldn't checks default to enabled, not disabled?